python-package/xgboost/spark/core.py

```diff
@@ -1112,20 +1188,9 @@ def _transform(self, dataset: DataFrame) -> DataFrame:
         features_col, feature_col_names = self._get_feature_col(dataset)
         enable_sparse_data_optim = self.getOrDefault(self.enable_sparse_data_optim)
```

Comment:
Is `enable_sparse_data_optim` used?

Reply:
Yeah, It will be used in the common framework, pls refer to https://github.com/dmlc/xgboost/pull/9292/files#diff-8900fd245992de81e0a7b8cb7fcd18c37bae4473e8672e9b11fd7010cac358cfR1198